### PR TITLE
Add `swapIndexes` method

### PR DESCRIPTION
**Pull request by brunoocasali**
Add a new method in the Client `client->swapIndexes(array $indexes)` to swap indexes. The array should follow this structure: `[['indexA', 'indexB'], ['indexC', 'indexD'], ...]`

### CODE REVIEW

File: src/Endpoints/Indexes.php [hunk 1]
```diff
@@ -153,6 +153,11 @@ public function delete(): array
         return $this->http->delete(self::PATH.'/'.$this->uid) ?? [];
     }
 
+    public function swapIndexes(array $indexes): array
```
Comment (norkunas) [inline src/Endpoints/Indexes.php:156]:
```suggestion
    /**
      * @param array{0: list<non-empty-string>, 1: list<non-empty-string>}
      */
    public function swapIndexes(array $indexes): array
```

Doc could help for phpstan :)

Comment (brunoocasali) [reply src/Endpoints/Indexes.php:159]: Amazing, thanks @norkunas 🎉

Comment (bidoubiwa) [reply src/Endpoints/Indexes.php:159]: >     * @param array{0: list<non-empty-string>, 1: list<non-empty-string>} $indexes

Not sure i understand the syntax, is it a type working with the following example?

```
[
  { indexes: ['a', 'b'] },
  { indexes: ['a', 'b'] }
]
```

Comment (norkunas) [reply src/Endpoints/Indexes.php:159]: was on my phone, probably misunderstood from the tests.
then it should be:
```
@param list<array{indexes: list<non-empty-string>}>
```

**Review by bidoubiwa — COMMENTED** [review]
Tests are failing

**Pull request merged**